server/tests/api_v4/authenticated/test_vlob_create.py

```diff
@@ -136,8 +138,8 @@ async def test_authenticated_vlob_create_bad_key_index(
         case "index_2_unknown":
             bad_key_index = 2
 
-        case unknown:
-            assert False, unknown
+        case _:
```

Comment:
the `assert False, unknown` is less readable than this `raise Exception(...)`, but is used really often in the test codebase (in every test parametrized with a `kind` parameter consisting of a string)

I won't fight against this change, but I'd rather keep simple the way of writing this check (writing test is already annoying enough ^^)

Reply:
I've followed this approach https://github.com/Scille/parsec-cloud/pull/6620#discussion_r1514623030
But now that you mentioned, it's probably better to stay consistent with the existing code base.

Reply:
Done